ilastik/shell/gui/ilastikShell.py

```diff
@@ -1764,7 +1856,11 @@ def ensureNoCurrentProject(self, assertClean=False):
                 message += "(Unsaved changes in: {})".format(", ".join(dirtyApplets))
                 buttons = QMessageBox.Yes | QMessageBox.Cancel
                 response = QMessageBox.warning(
-                    self, "Discard unsaved changes?", message, buttons, defaultButton=QMessageBox.Cancel
+                    self,
+                    "Discard unsaved changes?",
+                    message,
+                    buttons,
+                    defaultButton=QMessageBox.Cancel,
```

Comment:
not sure this really comes from the pre-commit hook. At least for me black would leave the formatting alone in the pre-commit hook. Might be good to revisit your settings there (I guess key here is the trailing comma, do you have any other plugin that might have added it automagically?)

Reply:
yep, I am sorry for the formatting, I messed up a bit, but now I don't seem to have this issue. I will revert it back as well.